### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by elberthcorniell**
In this pull request I want to merge the next features:

- Responsive design for display grids.
- Add the original site Font Family.
- Use the same colors for titles.
- Sticky navigation bar.
- Other features.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

### Hi @elberthcorniell 

:+1: You have a very nice project and it's obvious you've put in a lot of work

However, there are some things you have to fix before this project can be approved.

## [REQUIRED]
- [x] On a medium screen, the TWN logo should appear at the leftmost part of the navigation bar
- [x] On a small screen, navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to seach icon)
- [x] The top three articles section should contain 3 articles with an image as a background.
- [x] The latest news and latest funding sections should be centred

## Work on the required changes and submit for another review

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

> * [x]  On a medium screen, the TWN logo should appear at the leftmost part of the navigation bar
> * [x]  On a small screen, navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon)
**The top three articles should be `<article>`s with an image as a background.**
- [ ]  The top three articles section should contain 3 articles with an image as a background.
> * [x]  The latest news and latest funding sections should be centered

### [OPTIONAL]  :yellow_circle:  

- [ ] Rename the branch for something more descriptive and 

- [ ] Latest funding is not showing as original, it changes from 4 elements in desktop to 3 in tablet and mobile

- [ ] All images should have a short description as an alternative text instead of the `#` symbol.

### [OPTIONAL] 
- Use the browsers media query inspector to observe the changes the original site has in different screensizes.
## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making most of the requested changes and getting this project approved. :+1:

- [OPTIONAL]: The footer shouldn't change through breakpoints. However, your breaks on the mobile screen size. Please fix this
elberthcorniell github io_tnw-clone_

You can merge this branch into the master branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as: **Phillip Musiime**

**Pull request merged**
